app/javascript/src/components/editor/Modals/CreateProjectModal.svelte

```diff
@@ -1,7 +1,7 @@
 <script>
   import Modal from "./Modal.svelte"
   import { currentProject, isSignedIn, modal } from "../../../stores/editor"
-  import { createProject, renameCurrentProject } from "../../../utils/project"
+  import { createProject, renameCurrentProject, setUrl } from "../../../utils/project"
```

Comment:
Pretty sure this doesn't need to be here?

Reply:
It does, there was a missing import